### ISSUE TITLE
Pro 7: Update Clock: Countdown to Time

### ISSUE THREAD

**Issue by tacvideo** [bitfocus/companion-module-renewedvision-propresenter#92]
The function to Update a Clock to a specific Time is not working under Pro 7 - works fine for 6 - I've tried different formats, no change
<img width="783" alt="Screen Shot 2021-12-02 at 1 54 12 PM" src="https://user-images.githubusercontent.com/73620763/144501570-7e6555a0-8125-4043-9280-39cca7cb871f.png">
.

**Comment by tacvideo** [comment]
Note that I can rename the Timer or change the Type, just can't seem to change the actual Time to count down to...

**Comment by tacvideo** [comment]
After more testing, found this to be a problem on Windows and Mac versions of 7.7.1, and I've noticed that whatever I set the clock to (as above), this action happens just fine. The problem is when you "Show Message" - whatever the Timer is set to in the Message overrides the "linked" Timer you just set.

i.e. this is more of an FYI / Heads up on PP7, not a bug in Companion... unless I'm missing something
